kuma/static/js/wiki-skinny.js

```diff
@@ -0,0 +1,724 @@
+(function(win, doc, $) {
+    'use strict';
+
+    /*
+        Togglers within articles (i.e.)
+    */
+    $('.toggleable').mozTogglers();
+
+    /*
+        Toggle for quick links with nested lists
+    */
+    (function() {
+        // Set up the quick links with the toggler
+        var $quickLinks = $('#quick-links');
+        setupTogglers($quickLinks.find('> ul > li, > ol > li'));
+        $quickLinks.find('.toggleable').mozTogglers();
+    })();
+
+    /*
+        Set up the zone subnav accordion
+    */
+    $('.subnav').each(function() {
+        var $subnav = $(this);
+        var $subnavList = $subnav.find(' > ol');
+        var minHeightFn = $('.zone-landing-header-preview-base').length ? setMinHeight : noop;
+
+        if (!$subnavList.length) {
+            return; // Exit if the subnav isn't set up properly
+        }
+
+        // Set the list items as togglers where needed
+        setupTogglers($subnavList.find('li'));
+
+        // Make them toggleable!
+        $subnavList.find('.toggleable').mozTogglers({
+            slideCallback: minHeightFn
+        });
+
+        // Try to find the current page in the list, if found, open it
+        // Need to keep track of the elements we've found so they aren't found twice
+        var used = [];
+        var $selected = $subnavList.find('a[href$="' + doc.location.pathname + '"]');
+        $selected.each(function() {
+            var self = this;
+            var $togglers = $(this).parents('.toggleable').find('.toggler');
```

Comment:
Seeing that you cached `this` in `self` above, perhaps change this to `$(self).`